### PR TITLE
Create a VM on an Azure Arc-enabled vSphere cluster

### PR DESCRIPTION
**Pull request by lanicolas**
New scenario to create a Windows VM with an ARM template that will run on an Azure Arc-enabled vSphere cluster using custom locations

### CODE REVIEW

File: docs/azure_arc_jumpstart/azure_arc_vsphere/day2/vmware_arm_template_win/_index.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    ```json
+    {
+    "appId": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "displayName": "JumpstartArc",
+    "password": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "tenant": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+    }
+    ```
+
+    > **NOTE: It is optional but highly recommended to scope the SP to a specific [Azure subscription and resource group](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest)**
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_vsphere/day2/vmware_arm_template_win/_index.md:60]:
```suggestion
    > **NOTE: It is optional but highly recommended to scope the SP to a specific [Azure subscription and resource group](https://docs.microsoft.com/cli/azure/ad/sp?view=azure-cli-latest)**
```

File: docs/azure_arc_jumpstart/azure_arc_vsphere/day2/vmware_arm_template_win/_index.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+## Create the Virtual Machine
+
+As mentioned, this deployment will leverage an ARM template. You will deploy a single ARM template at resource group scope.
+
+- Before deploying the ARM template, login to Azure using AZ CLI with the ```az login``` command.
+
+- The deployment will use an ARM template parameters file to customize your environment. Before initiating the deployment, edit the [_azuredeploy.parameters.json_](https://github.com/microsoft/azure_arc/blob/main/azure_arc_vsphere_jumpstart/vmware_arm_template_win/azuredeploy.parameters.json) file located in your local cloned repository folder. Example parameters files are located [here](https://github.com/microsoft/azure_arc/blob/main/azure_arc_vsphere_jumpstart/vmware_arm_template_win/azuredeploy.example.parameters.json). Fill out the parameters according to your environment:
+
+  - _`virtualMachineName`_ - the name of the Windows virtual machine that will be created
+  - _`extendedLocationName`_ - the name of your custom location
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_vsphere/day2/vmware_arm_template_win/_index.md:81]:
```suggestion
  - _`extendedLocationName`_ - the name of your Azure Arc custom location
```

File: docs/azure_arc_jumpstart/azure_arc_vsphere/day2/vmware_arm_template_win/_index.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+  - _`vCenterName`_ - the name of your vCenter
+  - _`networkName`_ - the network you will use to deploy your VM
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_vsphere/day2/vmware_arm_template_win/_index.md:83]:
```suggestion
  - _`networkName`_ - the vSphere network (port group) you will use to deploy your VM
```

File: azure_arc_vsphere_jumpstart/vmware_arm_template_win/azuredeploy.example.parameters.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+      "value": "Arc-VMware-Demo"
+    },
+    "extendedLocationName": {
+      "value": "jumpstart-cl"
+    },
+    "vCenterName": {
+      "value": "jumpstart-vCenter"
+    },
+    "networkName": {
+      "value": "Nested-Jumpstart-VM-Network-PG-VLAN-110-"
```
Comment (likamrat) [inline azure_arc_vsphere_jumpstart/vmware_arm_template_win/azuredeploy.example.parameters.json:15]:
```suggestion
      "value": "Nested-Jumpstart-VM-Network-PG-VLAN-110"
```

File: docs/azure_arc_jumpstart/azure_arc_vsphere/day2/vmware_arm_template_win/_index.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+  - _`nicType`_ - the network type to associate to the VM
+  - _`templateName`_ - your Windows VM template name
+  - _`clusterName`_ - the vSphere cluster to host the VM
+  - _`adminUserName`_ - the admin user name for the created Windows VM
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_vsphere/day2/vmware_arm_template_win/_index.md:87]:
```suggestion
  - _`adminUserName`_ - the admin username for the created Windows VM
```

File: docs/azure_arc_jumpstart/azure_arc_vsphere/day2/vmware_arm_template_win/_index.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+  - _`adminUserPassword`_ - the admin user password for the created Windows VM
+
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_vsphere/day2/vmware_arm_template_win/_index.md:89]: Please update where applicable in the first VMware scenario

**Review by likamrat — CHANGES_REQUESTED** [review]
Minor description suggestions

**Review by likamrat — CHANGES_REQUESTED** [review]
Minor description suggestions

**Comment by lanicolas** [comment]
> Minor description suggestions

Applied changes and aligned variable with the Resource Bridge scenario for VMware

**Review by likamrat — APPROVED** [review]
LGTM. Great work!!

**Pull request merged**